installation/installing-images/README.md

```diff
@@ -40,3 +40,12 @@ For more advanced control of this process, see our system-specific guides:
 - [Mac OS](mac.md)
 - [Windows](windows.md)
 - [Chrome OS](chromeos.md)
+
+
+## Boot your new OS
+
+You can now insert the SD card into the Raspberry Pi and power it up. 
+
+For the official Raspbian OS, if you need to manually log in, the default user name is `pi`, with password `raspberry`. Remember the default keyboard layout is set to UK.
+
+You should change the default password straight away to ensure your Raspberry Pi is secure.
```

Comment:
Ubuntu for the Pi forces the user to change the primary user password at first log on. Is that something Raspbian should be doing? (Not a question for the documentation, I know).

Reply:
For the audience that Raspbian is aimed at, I suspect enforcing a mandatory password change might cause more problems than it "fixes"? See also Simon's earlier comments above...

Reply:
Perhaps it's worth linking the word "secure" in the sentence above to https://github.com/raspberrypi/documentation/blob/master/configuration/security.md ? :man_shrugging:

Reply:
Ah, good idea.